dev-packages/browser-integration-tests/suites/integrations/httpclient/fetch/withAbortController/test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from '@playwright/test';
+import type { Event as SentryEvent } from '@sentry/types';
+import { sentryTest } from '../../../../../utils/fixtures';
+import { getFirstSentryEnvelopeRequest } from '../../../../../utils/helpers';
+
+sentryTest('should handle aborted fetch calls', async ({ getLocalTestPath, page }) => {
+  const url = await getLocalTestPath({ testDir: __dirname });
+
+  await page.route('**/foo', async route => {
+    setTimeout(() => {
+      route.fulfill({
+        status: 200,
+        contentType: 'application/json',
+        body: JSON.stringify({ message: 'This is the server response' }),
+      });
+    }, 2000);
```

Comment:
I would probably never fulfill this. It has potential to flake otherwise.

Reply:
Wouldn't this run into quite a long timeout then?

Reply:
the request is abortet in any case so nothing should time out.